proto/substreams/sink/database/v1/database.proto

```diff
@@ -22,6 +25,10 @@ message TableChange {
   repeated Field fields = 5;
 }
 
+message CompositePrimaryKey {
+  map<string, string> keys = 1;
```

Comment:
Is the ordering of the composite parts important?

Reply:
For performance, it's important while creating your primary key. For example:
if you have a primary key (A, B) and you usually query by B the ordering is important and should be (B, A).

If you are filtering both, it isn't important.

Reply:
Ok so the safest thing you be to use an ordered container then. The `map<string, string>` has no guaranteed ordering. So it would be better to have something like if I understand you correctly:

```suggestion
  message Key {
    string name = 1;
    string value = 2;
  }

  repeated Key keys = 1;
```

?

Reply:
Perfect!